### ISSUE TITLE
Access Policy for Blob Index Tags

### ISSUE THREAD

**Issue by freeloop914** [Azure/azure-cli#20452]
**Is your feature request related to a problem? Please describe.**
This is an extended request from #20368 

```
az storage container policy create --name "mypolicy" --container-name "mycontainer" --account-name "myaccount"  --permissions wt

valid values are (a)dd (c)reate (d)elete (l)ist (r)ead (w)rite or a combination thereof.
```

**Describe the solution you'd like**
Add `t` `f` to acceptable values for --permissions.

**Additional context**
az powershell works good
```
$SharedAccessPolicy = @{
    Context = $context
    Policy = "mypolicy"
    Container = "mycontainer"
    ExpiryTime = (Get-Date).ToUniversalTime().AddYears(1)
    Permission = "wt"
}
New-AzStorageContainerStoredAccessPolicy @SharedAccessPolicy
```

**Comment by yonzhan** [comment]
storage

**Comment by jiasli** [comment]
@freeloop914, sorry but I don't get your point about "extended" - is any **additional** feature needed compared to #20368?

**Comment by freeloop914** [comment]
> @freeloop914, sorry but I don't get your point about "extended" - is any **additional** feature needed compared to #20368?

I mean #20368 only mentioned `az storage account generate-sas`.
I created this feature request is for `az storage container policy`

**Comment by freeloop914** [comment]
@jiasli 
Another issue:
I used azure powershell to set access policy with 'wt' permission.
Then I used `az storage container policy list` to get the list and it returns error.

`
$ az storage container policy list -c internal
ERROR: Client-Request-ID=2abdb54d-6794-11ec-9581-c8348e334d1c Retry policy did not allow for a retry: Server-Timestamp=Tue, 28 Dec 2021 04:11:00 GMT, Server-Request-ID=d27352ac-b01e-002e-66a0-fbff8c000000, HTTP status code=409, Exception=Stored access policy contains a permission that is not supported by this version. ErrorCode: FeatureVersionMismatch<?xml version="1.0" encoding="utf-8"?><Error><Code>FeatureVersionMismatch</Code><Message>Stored access policy contains a permission that is not supported by this version.RequestId:d27352ac-b01e-002e-66a0-fbff8c000000Time:2021-12-28T04:11:01.7761785Z</Message></Error>.
ERROR: Stored access policy contains a permission that is not supported by this version. ErrorCode: FeatureVersionMismatch
<?xml version="1.0" encoding="utf-8"?><Error><Code>FeatureVersionMismatch</Code><Message>Stored access policy contains a permission that is not supported by this version.
RequestId:d27352ac-b01e-002e-66a0-fbff8c000000
Time:2021-12-28T04:11:01.7761785Z</Message></Error>

`
